### PR TITLE
Update team photos

### LINKED ISSUE #542: Change avatar photos
The avatars are blue at the moment, which is ironically right, but in this case, not.

### PR DESCRIPTION
**Pull request by jreis13**
Why:

- Closes #542 

These changes address the need by:

- Delete unused `horizontal` from Photo
- Delete horizontal versions of avatars
- Replace vertical for avatar
- Update team photos
- Add photos from other team members

### CODE REVIEW

File: config/node/createSchemaCustomization.js [hunk 1]
```diff
@@ -18,8 +18,7 @@ module.exports = ({ actions }) => {
       }
 
       type Photo {
-        horizontal: File! @fileByRelativePath,
-        vertical: File! @fileByRelativePath,
+        avatar: File! @fileByRelativePath,
```
Comment (mirhamasala) [inline config/node/createSchemaCustomization.js:21]: Should we place this directly under the `Author` type? If there's just one photo anyway? CC: @gabrielpoca

Comment (gabrielpoca) [reply config/node/createSchemaCustomization.js:21]: Yeah, I think so. I don't see why we need this Photo type.

Comment (mirhamasala) [reply config/node/createSchemaCustomization.js:21]: @jreis13 Will you take care of this?

Comment (mirhamasala) [reply config/node/createSchemaCustomization.js:21]: When that's done, we can review again and merge this. Then, people can check their photo live, and request a change if they want.

**Review by mirhamasala — APPROVED** [review]

**Pull request merged**